src/app/space/create/codebases/codebases-item-heading/codebases-item-heading.component.html

```diff
@@ -22,4 +22,12 @@
       </div>
     </div>
   </div>
+  <div class="list-pf-additional-content">
+    <div>
```

Comment:
This `<div>` is not necessary, as the content is already wrapped in a `<div>`.

Reply:
This div helped to have title and icon in sync if I remove it, may need to override some styles. Without that `<div>` it's as below

<img width="346" alt="screen shot 2018-07-07 at 10 14 12 am" src="https://user-images.githubusercontent.com/5129024/42406893-d50221f4-81ce-11e8-980d-9396ea85c45d.png">